app/_landing_pages/terraform.yaml

```
@@ -1,18 +1,218 @@
+      - blocks:
+        - type: card
+          config:
+            icon: /assets/icons/mesh.svg
+            title: Konnect (Beta)
+            description: |
+              Konnect regularly ships new features, and they start out as a _beta_ release. In order to provide early access to these features using Terraform, we provide the `konnect-beta` provider.
+
+              The **beta** provider currently supports:
+
+              * Kong Mesh
+              * Portal v3
+            ctas:
+              - text: GitHub
+                url: "https://github.com/Kong/terraform-provider-konnect-beta"
+              - text: Terraform Registry
+                url: "https://registry.terraform.io/providers/Kong/konnect-beta/latest"
+              - text: Examples
+                url: "https://github.com/Kong/terraform-provider-konnect-beta/tree/main/examples/scenarios"
+      - blocks:
+        - type: card
+          config:
+            icon: /assets/icons/mesh.svg
+            title: "{{site.base_gateway}}"
+            description: |
+              Designed for use with {{site.base_gateway}} on-prem, the `kong-gateway` provider allows you to manage Kong Entities using Terraform
+
+              This provider is currently available as a **BETA** release
+            ctas:
+              - text: GitHub
+                url: "https://github.com/Kong/terraform-provider-kong-gateway"
+              - text: Terraform Registry
+                url: "https://registry.terraform.io/providers/Kong/kong-gateway/latest"
+              - text: Examples
+                url: "https://github.com/Kong/terraform-provider-kong-gateway/tree/main/examples/scenarios"
+      - blocks:
+        - type: card
+          config:
+            icon: /assets/icons/mesh.svg
+            title: Kong Mesh
+            description: |
+              The `kong-mesh` provider allows you to manage Mesh policies in an on-prem deployment of Kong Mesh
+
+              This provider is currently available as a **BETA** release
+            ctas:
+              - text: GitHub
+                url: "https://github.com/Kong/terraform-provider-kong-mesh"
+              - text: Terraform Registry
+                url: "https://registry.terraform.io/providers/Kong/kong-mesh/latest"
+              - text: Examples
+                url: "https://docs.konghq.com/mesh/latest/guides/deploy-kong-mesh-using-terraform-and-konnect/"
+
+  - header:
+      type: h2
+      text: "Sample Workflows"
+      sub_text: |
+        The GitHub repositories for each provider contain pre-built scenarios.
+
+        Here are a couple of examples of what's possible:
+    columns:
+      - blocks:
+        - type: structured_text
+          config:
+            header:
+              text: "Rate Limit a Service"
+            blocks:
+              - type: text
+                text: |
+                  This is a longer description
```

```suggestion
                 Deploy a Gateway Control Plane with a Gateway Service, Route, and a rate limiting policy of 5 requests per minute on the Route.
```
Not sure if this is what you were going for, but to have something.